requirements.txt

```diff
@@ -1,6 +1,6 @@
 numpy==1.23.4
 pandas==1.4.2
-python-Levenshtein==0.12.0
+levenshtein==0.20.9
```

Comment:
@ayyubibrahimi this is the culprit. `datamatch` is dependent on python-Levenshtein v0.12.0. I wonder why this dependency is updated in the first place.

Reply:
If we don't update that dependency, we can't build our docker image. Have you recently tried to rebuild the docker image with `python-Levenshtein==0.12.0`?

Reply:
Actually they seem to be 2 separate packages, so they can probably coexist. Can you try putting back in the python-levenshtein package?

Reply:
If that doesn't work I'll update datamatch. It should work because both packages seem to be just different versions.

Reply:
Thanks Khoi. I'm not near my laptop atm but I'll update it ASAP.

Reply:
I reverted it back to the original version. I also had to clean up the cache in the docker builder. It's successfully built the docker image.